### ISSUE TITLE
[Feature Request] suppress newlines

### ISSUE THREAD

**Issue by r0bj** [kelseyhightower/confd#485]
It would be great to have ability to control the newlines output by the template, eg.:
`{{-end}} // eats leading whitespace`

I think it's implemented in go text/template:
https://github.com/golang/go/issues/9969

**Comment by cpoole** [comment]
This exists. You need go 1.6 or higher. if you use the 0.12.0-alpha3 release this feature works

**Comment by bacongobbler** [comment]
That is correct. I believe the current releases have been compiled with earlier go releases so you'll need to compile from scratch to achieve this functionality.

**Comment by r0bj** [comment]
You are right, I've compiled confd with go 1.6 and it's working as expected.
Thanks.
